doc/changelogs/CHANGELOG_V10.md

```diff
@@ -59,6 +60,95 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="10.20.0"></a>
+## 2020-04-07, Version 10.20.0 'Dubnium' (LTS), @BethGriggs
+
+### macOS package notarization and a change in builder configuration
+
+The macOS binaries for this release, and future 10.x releases, are now
+being compiled on macOS 10.15 (Catalina) with Xcode 11 to support
+package notarization, a requirement for installing .pkg files on macOS
+10.15 and later. Previous builds of Node.js 10.x were compiled on macOS
+10.7 (Lion). As binaries are still being compiled to support a minimum
```

Comment:
I think (https://github.com/nodejs/build/commit/050fb9733697acb3dd5bb53891c2818e2c335bab#diff-41fc98d41f1701391fd5a43db36b75d0) we previously built Node.js 10.x on 10.10 (Yosemite) with a minimum deployment target of macOS 10.7 (Lion).

Reply:
Ah!

Reply:
Updating to: 
> ### macOS package notarization and a change in builder configuration

> The macOS binaries for this release, and future 10.x releases, are now
being compiled on macOS 10.15 (Catalina) with Xcode 11 to support
package notarization, a requirement for installing .pkg files on macOS
10.15 and later. Previous builds of Node.js 10.x were compiled on macOS
10.10 (Yosemite) with a minimum deployment target of macOS 10.7 (Lion).
As binaries are still being compiled to support a minimum of macOS 10.7
(Lion) we do not anticipate this having a negative impact on Node.js 
10.x users with older versions of macOS.

Reply:
SGTM